src/libs/workspace-utils.ts

```diff
@@ -56,7 +56,7 @@ export const isAzureWorkspace = (workspace: BaseWorkspace): workspace is AzureWo
 }
 
 export const isGoogleWorkspace = (workspace: BaseWorkspace): workspace is GoogleWorkspace => {
-  return isGoogleWorkspaceInfo(workspace.workspace)
+  return workspace && isGoogleWorkspaceInfo(workspace.workspace)
```

Comment:
I don't like adding the check on `workspace` here. You should only call this method if the workspace is already loaded.

The reason is that a lot of times folks assume if `isGoogleWorkspace` is false, then the workspace is Azure (or at least not Google). But if workspace is undefined, you really have no idea what it is.

Reply:
Also, once callers of this method are in TypeScript, I don't think it will be valid to call `isGoogleWorkspace` with something that is undefined.

Reply:
The problem is that, if this method is called from a typescript file, this method properly typechecks and tells you that you cannot call it with a workspace that could be undefined. However, if it is being called from a non-typescript file, we don't have that typesafety, and you get a runtime error if your workspace isn't defined when calling this. 

The relevant call site does check `workspace` though, so I'll remove it.

Reply:
I think the runtime error is correct-- better than causing potentially confusing behavior.